app/models/partner_distribution.rb

```diff
@@ -0,0 +1,28 @@
+class PartnerDistribution
+  include Exportable
+
+  def self.for_csv_export(organization, partner_id:)
```

Comment:
This is the part that makes this WIP. The `partner_id` is currently not passed. How that is ends up working depends on the implementation for https://github.com/rubyforgood/diaper/issues/1870.